software/control/microscope.py

```diff
@@ -179,4 +180,149 @@ def close(self):
         self.camera.close()
         self.microcontroller.close()
         if USE_ZABER_EMISSION_FILTER_WHEEL or USE_OPTOSPIN_EMISSION_FILTER_WHEEL:
-            self.emission_filter_wheel.close()
\ No newline at end of file
+            self.emission_filter_wheel.close()
+
+
+
+class LightSourceType(Enum):
+    Squid = 0
+    LDI = 1
+    CELESTA = 2
+    VersaLase = 3
+    SCI = 4
+
+class IntensityControlMode(Enum):
+    OnBoardDAC = 0
+    Software = 1
+    USB = 2
```

Comment:
We should remove USB - it's the same as software (can be ethernet instead of USB)